src/renderer/components/watch-video-info/watch-video-info.js

```diff
@@ -105,11 +105,11 @@ export default defineComponent({
     }
   },
   computed: {
-    hideSharingActions: function() {
+    hideSharingActions: function () {
```

Comment:
Please revert unnecessary space changes

Reply:
I'm sorry, that must have been my auto-formatting. 

Fixed it in the latest commit 👍